superset/reports/commands/execute.py

```diff
@@ -77,11 +77,22 @@
 logger = logging.getLogger(__name__)
 
 
-def _get_user() -> User:
-    user = security_manager.find_user(username=app.config["THUMBNAIL_SELENIUM_USER"])
-    if not user:
-        raise ReportScheduleSelleniumUserNotFoundError()
-    return user
+def _get_user(report_schedule: ReportSchedule) -> User:
+    user_types = app.config["ALERT_REPORTS_EXECUTE_AS"]
+    for user_type in user_types:
+        if user_type == "selenium":
+            return app.config["THUMBNAIL_SELENIUM_USER"]
+        if user_type == "creator":
+            if user := report_schedule.created_by:
```

Comment:
@villebro the walrus operator isn't supported in Python 3.8.

Reply:
Hmm, I do believe it is: https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions